server/impl/user_helpers_test.go

```diff
@@ -0,0 +1,31 @@
+package impl_test
+
+import (
+	"fmt"
+	"sync/atomic"
+
+	"gorm.io/gorm"
+
+	"go.timothygu.me/downtomeet/server/db"
+)
+
+var emailNumber int64 = 0
+
+func newEmail() string {
+	num := atomic.AddInt64(&emailNumber, 1)
+	return fmt.Sprintf("%v@email.com", num)
+}
+
+func createUser() *db.User {
+	newUser := db.User{
+		Model:       gorm.Model{},
+		Email:       newEmail(),
+		Name:        newEmail(),
+		ContactInfo: "",
+		Location:    db.Coordinates{},
```

Comment:
```suggestion
		Email:       newEmail(),
		Name:        newEmail(),
```

Don't specify things you don't need.